packages/web/server/queries/osmosis/concentratedliquidity/position-by-id.ts

```diff
@@ -0,0 +1,19 @@
+// "/osmosis/concentratedliquidity/v1beta1"/positions/${bech32Address}?pagination.limit=10000
```

Comment:
Are these comments for others' reference?

Reply:
I overlooked this earlier. It has been removed in my latest commit.